### PR TITLE
Fix xml docs.

### PR DESCRIPTION
**Pull request by surfnerd**
fix xml doc validation.

### CODE REVIEW

File: com.unity.ml-agents/Runtime/Policies/BehaviorParameters.cs [hunk 1]
```diff
@@ -74,9 +74,16 @@ public class BehaviorParameters : MonoBehaviour
...
+        /// Delegate for receiving events about Policy Updates.
+        /// </summary>
+        /// <param name="isInHeuristicMode">Whether or not the current policy is running in heuristic mode.</param>
         public delegate void PolicyUpdated(bool isInHeuristicMode);
 
-        internal event PolicyUpdated OnPolicyUpdated;
+        /// <summary>
+        /// Event that fires when an Agent's policy is updated.
+        /// </summary>
+        public event PolicyUpdated OnPolicyUpdated;
```
Comment (dongruoping) [inline com.unity.ml-agents/Runtime/Policies/BehaviorParameters.cs:86]: is it intended to make it public?

Comment (surfnerd) [reply com.unity.ml-agents/Runtime/Policies/BehaviorParameters.cs:86]: yeah

Comment (surfnerd) [reply com.unity.ml-agents/Runtime/Policies/BehaviorParameters.cs:86]: ah crap, i can't

Comment (surfnerd) [reply com.unity.ml-agents/Runtime/Policies/BehaviorParameters.cs:86]: 😢

**Review by dongruoping — APPROVED** [review]

**Pull request merged**